test/test_retryable_writes.py

```diff
@@ -581,8 +602,56 @@ def test_pool_paused_error_is_retryable(self):
         failed = cmd_listener.failed_events
         self.assertEqual(1, len(failed), msg)
 
+    @client_context.require_failCommand_fail_point
+    @client_context.require_replica_set
+    @client_knobs(heartbeat_frequency=0.05, min_heartbeat_interval=0.05)
+    def test_returns_original_error_code(
+        self,
+    ):  # TODO: Make this a real integration test where we stepdown the primary.
```

Comment:
Was this TODO meant to be here?  It looks like it was referring to the TestRetryableWritesTxnNumber class.